test/regress_util.c

```diff
@@ -1536,6 +1540,169 @@ test_evutil_v6addr_is_local(void *arg)
 	;
 }
 
+#define SOCKETPAIR_CLOSE(fd)	do {	\
+	if (fd[0] != -1) evutil_closesocket(fd[0]);	\
+	if (fd[1] != -1) evutil_closesocket(fd[1]);	\
```

Comment:
Ugh, why this is a macro, not a static function? (don't see anything that requires to be run by preprocessor)